### ISSUE TITLE
One command to 

### ISSUE THREAD

**Issue by ddeboer** [FriendsOfSymfony/FOSHttpCacheBundle#203]
Some cached responses need to be invalidated (only) when the application is deployed. We now have [separate commands]
(http://foshttpcachebundle.readthedocs.org/en/latest/features/invalidation.html#console-commands) that do cache invalidation. However, it’s easier to just run one command from the deploy scripts. We suggest users write their own command if they need more advanced stuff. 

Would it make sense to offer a little help here, by 
* adding one `fos:httpcache:invalidate` (e.g.) command with a `config` argument that refers to a config key
* adding the possibility to combine groups of routes in and path under a certain key `config.yml`.

Does this fall into the scope of this bundle?

**Comment by dbu** [comment]
this only works when doing the rules in configuration. not sure if that is flexible enough. but having an additional groups attribute there and being able to reference those in a command sounds sensible. maybe we can show a way to add custom things to groups... trigger an event maybe? for those cases where configuration is not enough.

**Comment by ddeboer** [comment]
Right, so we would start with adding groups/tags to invalidation config rules (and annotations!), and a way to retrieve all config rules/annotations for a group. I guess it’s best to introduce a new service that does the retrieval.

**Comment by dbu** [comment]
agreed.
